src/content/docs/infrastructure/host-integrations/host-integrations-list/stripe-integration.mdx

```diff
@@ -0,0 +1,125 @@
+To set up synthetic monitors, follow the steps in our [guided install](https://onenr.io/0EjOXrnxgQ6)
+
+## Send metrics to New Relic
+Check the [Metric API documentation](https://docs.newrelic.com/docs/data-apis/ingest-apis/metric-api/report-metrics-metric-api/) for further details.
+
+To proceed further, you will need a New Relic license key. Kindly follow the [New Relic API key documentation](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/) to get your license key.
+On a successful stripe payment, we receive a JSON from the stripe API service.
+We need to send these values to New Relic by formatting the JSON as per New Relic Metric API requirements.
+
+Below is the javascript code snippet which formats the data and sends it to New Relic metric API URL
+```
+const sendDataToNewRelic = (sessionJSON) => {
+  const currentDate = new Date();
+  const currentTime = currentDate.getTime();
+  const nrMetrics = [{
+    "metrics": [
+      {
+        "name": "stripe_metrics",
+        "timestamp": currentTime,
+        "value": 0,
+        "attributes": {
+          "stripe.session_id": sessionJSON.id,
+          "stripe.amount_total": sessionJSON.amount_total,
+          "stripe.customer_email": sessionJSON.customer_details.email,
+          "stripe.created_at": sessionJSON.created,
+          "stripe.currency" : sessionJSON.currency,
+          "stripe.payment_status": sessionJSON.payment_status,
+          "stripe.status": sessionJSON.status,
+          "stripe.discount_amount":sessionJSON.total_details.amount_discount,
+          "stripe.shipping_amount":sessionJSON.total_details.amount_shipping,
+          "stripe.tax_amount":sessionJSON.total_details.amount_tax
+        }
+      }
+    ]
+  }]
+
+  const nrMetricAPI = 'https://metric-api.newrelic.com/metric/v1'
+  axios({
+    url: nrMetricAPI,
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded',
+      'Api-Key': 'LICENSE_KEY'
+     },
+    data: JSON.stringify(nrMetrics)
+  })
+   .then((response) => {
+    // Handle the response here if needed
+    console.log('Metrics data sent successfully:', response.data);
+  })
+  .catch((error) => {
+    // Handle any errors that occur during the request
+    console.error('Error sending metrics data to New Relic:', error);
+  })
+}
+
+```
+**Note:** Change the sessionJSON according to your response and call this function where you are getting stripe metrics in your application.
+```
+ sendDataToNewRelic (session)
+```
+
+## Monitor your application
+You can choose our pre-built dashboard template named `Stripe` to monitor your Stripe application metrics.
+1. Go to **[one.newrelic.com](https://one.newrelic.com/)** and click on **+ Add data**.
+2.  Click on the **Dashboards** tab.
+3.  In the search box, type `Stripe`.
+4.  When you see our pre-build dashboard, click on it to install it in your account.
```

Comment:
@sjyothi54 I wasn't able to find this dashboard. Can you provide a direct link to it? 
<img width="1332" alt="image" src="https://github.com/newrelic/docs-website/assets/42753584/c9572b8e-b419-4853-8be4-d52890893a65">

Reply:
Stripe dashboard:https://onenr.io/0ZQWYYE1rRW